packages/ui/src/lib/fontSizes.ts

```diff
@@ -0,0 +1,33 @@
+import { CSSObject } from '@emotion/react'
+import { Level, mq } from '../lib/media-query'
+import { FontSizes, fontSizes } from './theme/typography'
+
+export type PartialRecord<K extends keyof any, T> = Partial<Record<K, T>>
+
+export type FontSizeProps = FontSizes | PartialRecord<Level | '_', FontSizes>
+
+export const getFontSize = (sizes: FontSizeProps) => {
+  let styles: CSSObject = {}
+  if (typeof sizes === 'number') {
+    styles = { fontSize: fontSizes[sizes] }
+  } else if (sizes) {
```

Comment:
Since this function expects has a required argument I believe it would be better not to check for the "presence" of `sizes` here but make sure of only call this function with a defined parameter. That would mean change here to:

```suggestion
  } else {
```

and update `Text` component like the following:

```
const TextElement = styled(
  'span',
  elementConfig,
)<TextProps>(({ theme, align, color, size = 'md' }) => ({
  color: color ? theme.colors[color] : 'inherit',
  ...(size && getFontSize(size)),
  ...(align && { textAlign: align }),
}))
```